components/next_steps_view/next_steps_tips.tsx

```diff
@@ -181,50 +194,76 @@ export default function NextStepsTips(props: Props) {
                         </button>
                     </div>
                 </Card>
-                <Card expanded={true}>
-                    <div className='Card__body'>
-                        <div className='Card__image'>
-                            <img
-                                src={tipsNextStepsImage2}
-                            />
-                        </div>
-                        <h3>
+                { props.isCloud ?
```

Comment:
These new steps will replace all current steps for Admins so no need to check for cloud